docs/README.md

```diff
@@ -49,7 +50,8 @@ If you find problems when you run MCinaBox, you can check [Q&A](./Q&A.md) first.
 ### Build
 - Build via Android Studio.
 
-#### You can also get released apk from [here](https://github.com/AOF-Dev/MCinaBox/releases).
+## Release
+- You can also get released apk from [here](https://github.com/AOF-Dev/MCinaBox/releases).
```

Comment:
User can find it

Reply:
so it's unless

Reply:
but its cool

Reply:
please this is my very first thing to try